python/pyhdk/hdk.py

```diff
@@ -2500,6 +2500,20 @@ def sql(self, sql_query, query_opts=None, **kwargs):
         res.scan = self.scan(res.table_name)
         return res
 
+    def clear_gpu_mem(self):
+        """
+        Clears GPU memory of all previously transferred buffers.
+        """
+        self._executor.clearMemory(self._data_mgr, 2)
+
+    def create_refragmented_view(self, table_name, refragmented_view_name, frag_size):
```

Comment:
To be more consistent with the rest of API, I propose a few things:
1. Make view name optional here
2. Return a scan node of the created view that can be used for queries
3. Accept both table_name and scan node as input
4. Add similar interface to the scan node to move it there completely
```
ht1 = hdk.import_csv(...)
hdk.sql("SELECT ... FROM t;", t = ht1)
ht2 = ht1.create_refragmented_view(1000000)
hdk.sql("SELECT ... FROM t;", t = ht2)
hdk.drop_table(ht1)
hdk.drop_table(ht2)
```